Resources/config/routing.yml

```diff
@@ -14,3 +14,8 @@ template_yui_module:
 ez_pjax:
     resource: "@eZPlatformUIBundle/Resources/config/routing_pjax.yml"
     prefix:   /pjax
+
+yui_combo_loader:
+    path: /combo
```

Comment:
from a performance perspective, it's better if we keep it short, `/_combo` or `/_ezcombo` or whatever :)